action/protocol/poll/governance_protocol.go

```diff
@@ -152,15 +152,23 @@ func (p *governanceChainCommitteeProtocol) CreatePreStates(ctx context.Context,
 		if err != nil {
 			return err
 		}
-		return setKickoutBlackList(sm, unqualifiedList)
+		return setNextEpochBlacklist(sm, unqualifiedList)
 	} else if blkCtx.BlockHeight == epochStartHeight && hu.IsPost(config.Easter, epochStartHeight) {
-		if err := shiftCandidates(sm); err != nil {
+		prevHeight, err := shiftCandidates(sm)
+		if err != nil {
 			return err
 		}
 		if blkCtx.BlockHeight == 1 {
```

Comment:
move the logic into createGenesisStates.

Reply:
fixed in latest commit